src/main/java/AddCommand.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Represents a command that executes the addition of a task into the task list.
+ */
+public class AddCommand extends Command {
+    protected Task task;
```

Comment:
Not a necessity but I recommend the changing of the visibility of this field to private. Still in the process of looking through your code, you may ignore this comment if the option is accessed in other classes of the same package.

Reply:
The same would apply for the other classes that inherit from `Command`. I've commented on two so far, but will not comment on the rest to reduce repetition.